misc/docusaurus/sidebars.js

```diff
@@ -102,6 +102,7 @@ const sidebars = {
                             label: 'std',
                             items: [
                                 'reference/standard-library/std/address',
+                                'reference/standard-library/std/realm',
```

Comment:
do we still have this file on master?

Reply:
This file was not present on master before. It was added just now.